### ISSUE TITLE
Corrupted Edge/Mara Bug

### ISSUE THREAD

**Issue by tionblack** [WarEmu/WarBugs#10366]
I am using Brute Strenght which gives 100 str in rvr with my marauder lvl 24 and corrupted Edge (100str/100 Init) yet i only get str from BRUTE STR not the 2nd tactic.

**Comment by Lheana** [comment]
Have you verified it using .getstats?

**Comment by Zealote** [comment]
Might be related to https://github.com/WarEmu/WarBugs/issues/717. Luth said in that report that it worked fine with Discipline, and I've just tried Brute Force - that also works correctly. However, I tested it with Marauder's Hulking Brute tactic to see if it was different for specced tactic. The results are below:

**Comment by Zealote** [comment]
Might be related to https://github.com/WarEmu/WarBugs/issues/717. Luth said in that report that it worked fine with Discipline, and I've just tried Brute Force - that also works correctly. However, I tested it on my level 21 Marauder with Hulking Brute (HB) tactic to see if it was different for a specced tactic. The toughness values, with the first being from char window and second from .getstats, are below:

In Warcamp, with no tacs equipped: Tou = 162 / 162
In Warcamp, with HB equipped:       Tou = 246 / 246 (+84, corresponds to tooltip)
In RvR, bolstered to 48, with HB **still** equipped: Tou = 313 / 310
In RvR, switched to no tacs: Tou = 313 / 310
In RvR, re-equipped HB: Tou = 397 / 394

Should be noted that when I tested going from no tactic in WC to no tactic in RvR (i.e. the only effect was bolster), the change to the toughness value was the same as with the "tac still equipped" case above: it went from 162 to 313 / 310.

Whether it's just an issue with Mara tactics or speccable ones in general, I don't know.
